### ISSUE TITLE
Angular 6 build : TypeError: Cannot read property 'kind' of undefined

### ISSUE THREAD

**Issue by anakin59490** [techiediaries/ngx-qrcode#64]
I'm getting the following error message when ng build --prod :   

![image](https://user-images.githubusercontent.com/16132965/90250806-fb13bc00-de3c-11ea-8ba4-dd359bf9cf1e.png)

## Environment

Angular version: 6.1.4

Version of `@techiediaries/ngx-qrcode`:
"@techiediaries/ngx-qrcode": "9.1.0",

 
For Tooling issues:
- Node version: 10.15.3
- Platform:  Windows

Others:
"devDependencies": {
    "@angular-devkit/build-angular": "~0.7.4",
    "@angular-devkit/build-ng-packagr": "~0.7.4",
    "@angular-devkit/core": "0.4.2",
    "@angular-devkit/schematics": "0.4.2",
    "@angular/cli": "^6.1.5",
    "@angular/compiler-cli": "6.1.4",
    "@angular/language-service": "6.1.4",

Thank you for your help

**Comment by alex88** [comment]
Same on angular 8

**Comment by HuangChunShui** [comment]
Same on angular6，solve by  "npm install ngx-qrcode2"

**Comment by Armando284** [comment]
Has anyone found a fix for this issue?

**Comment by Armando284** [comment]
> > Has anyone found a fix for this issue?
> 
> Did you find a solution for this???

No, actually I had to go back and use qrcode v1.5.1 and ngx-qrcode2 v0.0.9 since I can not update the project dependencies
It's quite similar to use
<ngx-qrcode
	[qrc-value]="couponCode"
	[qrc-element-type]="'canvas'"
	[qrc-class]="'qr-code'"
	#qrcElement
>
</ngx-qrcode>
